src/coins.h

```diff
@@ -188,17 +183,15 @@ struct CCoinsCacheEntry
     bool IsDirty() const noexcept { return m_flags & DIRTY; }
     bool IsFresh() const noexcept { return m_flags & FRESH; }
```

Comment:
we don't have Fresh anymore, only `IsDirtyAndFresh`, right?
```suggestion
    bool IsDirtyAndFresh() const noexcept
    {
        const bool is_fresh = m_flags & FRESH;
        Assume(IsDirty() || !is_fresh);
        return is_fresh;
    }
```